extensions/legit/porcelain.lisp

```diff
@@ -388,7 +621,7 @@ M	src/ext/porcelain.lisp
 (defun %maybe-lem-home ()
```

Comment:
Is it possible that the lem package does not exist?

Reply:
It is actually possible: I develop for Lem both in Lem and in Emacs. And in Emacs, I don't load Lem because… it is not clear to me how to do it, if I ever knew. Quickloading Lem isn't enough, one has to add its dependencies, etc.

So it's necessary for me that the porcelain package doesn't depend on Lem. And I think it's a good separation of concerns.

Reply:
Okay, I see why.
However, the dependency on lem is still there, just hidden and not statically obvious.
Ideally, porcelain.lisp should be loosely coupled to lem, but with this, wouldn't it be better to be able to resolve symbols statically with lem:lem-home?
That should be solved with just `(ql:quickload :lem)`.